shell/packages/sandstorm-db/db.js

```diff
@@ -56,6 +55,11 @@ if (Meteor.isServer && process.env.LOG_MONGO_QUERIES) {
 //       verifiedEmail: Only provided by some services. Cannot be directly edited by the user.
 //       main: True if this is the user's main identity.
 //       noLogin: True if the user does not trust this identity for account authentication.
+//       devName: Only present when `service` is "dev". A string containing the name used to create
+//                this dev account.
+//       emailTokens: Only present when `service` is "emailToken". An array of objects of the form
```

Comment:
Ugh. A problem with representing the service as capnp-style union like this:

```
"service" : {"emailToken": {"email": "dwrenshaw@gmail.com", "tokens": []}}
```

is that Mongo apparently doesn't make it easy to project out just the keys of an object. So we couldn't say "just return the service name" without explicitly specifying fields to exclude.

As such, I'm not super happy with this right now. I'll take another look in the morning.